grid.v3/registry/registry.go

```diff
@@ -55,6 +55,8 @@ type Registry struct {
 	keepAliveStats *keepAliveStats
 }
 
+var DefaultLeaseDuration = 60 * time.Second
```

Comment:
yea I wasn't a fan of doing this either but under the current design the registry is created by grid's server.Serve method and we don't have access to update the `LeaseDuration` before calling Serve.